src/app/player/directives/svg-icon.directive.ts

```diff
@@ -0,0 +1,15 @@
+import { Directive, ElementRef, Input, OnChanges, SimpleChanges } from '@angular/core';
+
+@Directive({
+  selector: '[svgIcon]'
+})
+export class SvgIconDirective implements OnChanges {
+  @Input()
```

Comment:
I think I understand that you are doing this because just setting `[innerHtml]` doesn't work since Angular sanitizes the passed html. 
Might I suggest a more general approach though, since that is really not only restricted to SVGs : a _general_ "safeInnerHtml" directive that sets that property on the native HtmlElement. Also, I'm not sure if that logic in `OnChanges` is really necessary. It could look something like this:

```
@Directive({
  selector: '[safeInnerHtml]'
})
export class SafeInnerHtmlDirective implements OnInit {
  @Input()
  safeInnerHtml?: string;

  constructor(private _elementRef: ElementRef) { }

  ngOnInit(): void {
    this._elementRef.nativeElement.innerHTML = this.safeInnerHtml;
  }
}
```

and used like: 

```
 <div class="player-container">
   <div [safeInnerHtml]="player?.avatar" class="svg"></div>
 </div>
```

Reply:
Yeah there are different solutions, I wanted for directive solution for aesthetic reason more than anything else (it looks better like that)

Reply:
done